CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 =====
 * [#81](https://github.com/kowainik/relude/issues/81):
   Add `asumMap` to `Foldable` functions.
+* [#89](https://github.com/kowainik/relude/issues/81):
+  Add `Relude.Extra.Type` module containing a `typeName` function.
```

Comment:
Could you also, please, update documentation in the `Relude` module here?

https://github.com/kowainik/relude/blob/25cada93f854399756a8f1d491bb7d7172dd0d98/src/Relude.hs#L78-L91

Just tell several words about the presence of `Relude.Extra.Type` module